detection_rules/schemas/definitions.py

```diff
@@ -33,7 +33,14 @@
     "allow_sample": (Version.parse('8.6.0'), None),
     "elasticsearch_validate_optional_fields": (Version.parse('7.16.0'), None)
 }
-NON_DATASET_PACKAGES = ['apm', 'auditd_manager', 'cloud_defend', 'endpoint', 'network_traffic', 'system', 'windows']
+NON_DATASET_PACKAGES = ['apm',
+                        'auditd_manager',
+                        'cloud_defend',
+                        'endpoint',
+                        'network_traffic',
+                        'system',
+                        'windows',
+                        'sentinel_one_cloud_funnel']
```

Comment:
@w0rk3r Is there any reason why we didn't use the [sentinel_one](https://github.com/elastic/integrations/tree/main/packages/sentinel_one) integration instead?

Reply:
It doesn't collect EDR telemetry, it focuses in the alerts and related data.